sys/include/net/ng_sixlowpan.h

```diff
@@ -10,6 +10,56 @@
  * @defgroup    net_ng_sixlowpan    6LoWPAN
  * @ingroup     net
  * @brief       6LoWPAN implementation
+ *
+ * This module is for usage with the @ref net_ng_netapi.
+ *
+ * # @ref net_ng_netapi command documentation
+ *
+ * ## `NG_NETAPI_MSG_TYPE_RCV`
+ *
+ * @ref NG_NETAPI_MSG_TYPE_RCV expects a packet in receive order
```

Comment:
I'm scratching my head because this describes the *behaviour* of -after following the link- a int value. Is this aiming at `ng_sixlowpan_print()`?